atomic_reactor/plugins/pre_fetch_maven_artifacts.py

```diff
@@ -56,7 +56,7 @@ def match(self, build_archive):
         return False
 
     def match_all(self, build_archives):
-        return filter(self.match, build_archives)
+        return list(filter(self.match, build_archives))
```

Comment:
Another formulation of this would be `[archive for archive in build_archives if self.match(archive)]`, which is at least more explicit.

Reply:
Ok, I like that